stdlib/public/runtime/Private.h

```diff
@@ -183,6 +183,8 @@ namespace swift {
                              void *data);
   uint8_t *_swift_getSectionDataPE(void *handle, const char *sectionName,
                                    unsigned long *sectionSize);
+#endif
+#if defined(__CYGWIN__)
   void _swift_once_f(uintptr_t *predicate, void *context,
```

Comment:
The definition of ```std::once_flag``` on MSVC is
```
struct once_flag
	{	// opaque data structure for call_once()
	constexpr once_flag() noexcept
		: _Opaque(0)
		{	// default construct
		}
     .....
	void *_Opaque;
	};
```
.
I verified that the variable of the type ```std::once_flag``` is initialized to zero and the variable is changed to 2 after I call ```std::call_once``` with it. Does this behavior match what we expect?